src/patternfly/components/Dropdown/dropdown-menu-item-title.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-dropdown__menu-item-title{{#if dropdown-menu-item-title--modifier}} {{dropdown-menu-item-title--modifier}}{{/if}}"
```

Comment:
I might just call this `menu-item-text` instead of `menu-item-title`

Reply:
Oh sorry I read the code before looking at the examples and assumed it just wrapped the text. Maybe `-main`?

Reply:
yeah that sounds good to me